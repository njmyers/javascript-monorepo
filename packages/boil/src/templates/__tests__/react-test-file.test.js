import reactTestFile from "../react-test-file";

describe("it works", () => {
  test("it is a function ", () => {
    expect(typeof reactTestFile).toBe("function");
  });

  test("it runs without crashing", () => {
    expect(reactTestFile("test-component")).toMatchSnapshot();
  });
});
