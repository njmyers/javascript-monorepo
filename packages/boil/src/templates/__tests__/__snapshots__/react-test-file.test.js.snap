// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works it runs without crashing 1`] = `
"import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import TestComponent from '../TestComponent';

describe('it works', () => {
  test('it loads without crashing', () => {
    const render = TestRenderer.create(<TestComponent />);
    expect(render.toJSON()).toMatchSnapshot();
  });
});

"
`;
